### PR TITLE
Updated models.py

### PR DESCRIPTION
**Pull request by kunalbhatia-868**
Moved import line to the top

### CODE REVIEW

**Comment by pmutua** [comment]
Hi @kunalbhatia-868 Thanks for contributing 👍🏽. The issues I added were to help someone else I'm reviewing to learn about Django & version control. I have some repos I would love to share with you if you like 😉.

File: catalog/models.py [hunk 1]
```diff
@@ -1,6 +1,11 @@
 from django.db import models
 from django.urls import reverse
 
+import uuid # Required for unique book instances
+from django.contrib.auth.models import Permission, PermissionsMixin, User
+from datetime import date
+
+
```
Comment (pmutua) [inline catalog/models.py:8]: Imports have been moved 👍🏽

**Review by pmutua — APPROVED** [review]
looks alright the imports have been moved

**Comment by kunalbhatia-868** [comment]
> Hi @kunalbhatia-868 Thanks for contributing 👍🏽. The issues I added were to help someone else I'm reviewing to learn about Django & version control. I have some repos I would love to share with you if you like 😉.

Are there any other Django projects I can contribute to . I have just started with open source and want to contribute to projects

**Pull request merged**